<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ShishirDwivedi" name="SeamlessStoreCard" tests="1" failures="1" timestamp="11 Nov 2014 12:45:36 GMT" time="55.637" errors="0">
  <testcase name="@AfterTest tearDown" time="0.0" classname="Test.AdminPanel.Filters.TestBlackListing">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
&amp;lt;font color=&amp;apos;Red&amp;apos;&amp;gt;Expected &amp;apos;Response Param &amp;apos;PG_TYPE&amp;apos;&amp;apos; was :-&amp;apos;&amp;apos;. But actual is &amp;apos;HDFCPG&amp;apos;&amp;lt;/font&amp;gt;&amp;lt;/br&amp;gt;This is generated by soft asset, &amp;lt;font color=&amp;apos;Red&amp;apos;&amp;gt;Failed to verify Test Response&amp;lt;/font&amp;gt;&amp;lt;/br&amp;gt;This is generated by soft asset, &amp;lt;font color=&amp;apos;Red&amp;apos;&amp;gt;java.lang.AssertionError: expected [true] but found [false]&amp;lt;/font&amp;gt;&amp;lt;/br&amp;gt;This is generated by soft asset, &amp;lt;font color=&amp;apos;Red&amp;apos;&amp;gt;&amp;lt;B&amp;gt;Failure occured in test &amp;apos;Test.AdminPanel.Filters.TestBlackListing.blackListedPhoneNoGlobal&amp;apos; Ended on &amp;apos;2014-11-11 18:15:35&amp;apos;&amp;lt;/B&amp;gt;&amp;lt;/font&amp;gt;&amp;lt;/br&amp;gt;This is generated by soft asset">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
<font color='Red'>Expected 'Response Param 'PG_TYPE'' was :-''. But actual is 'HDFCPG'</font></br>This is generated by soft asset, <font color='Red'>Failed to verify Test Response</font></br>This is generated by soft asset, <font color='Red'>java.lang.AssertionError: expected [true] but found [false]</font></br>This is generated by soft asset, <font color='Red'><B>Failure occured in test 'Test.AdminPanel.Filters.TestBlackListing.blackListedPhoneNoGlobal' Ended on '2014-11-11 18:15:35'</B></font></br>This is generated by soft asset
	at Utils.TestBase.tearDown(TestBase.java:132)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="blackListedPhoneNoGlobal" time="47.253" classname="Test.AdminPanel.Filters.TestBlackListing">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at Test.AdminPanel.Filters.TestBlackListing.blackListedPhoneNoGlobal(TestBlackListing.java:371)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 11 stack frames]]>
    </failure>
  </testcase> <!-- blackListedPhoneNoGlobal -->
</testsuite> <!-- SeamlessStoreCard -->
